sitestats/sitestats-bundle/src/resources/Messages.properties

```diff
@@ -325,3 +325,6 @@ predefined_report5_description = Show users who have never visited the site.
 #
 predefined_report6_title = Users with no activity
 predefined_report6_description = Show users with no activity in site.
+
+format_datetime = MM/dd/yyyy hh:mm a
+datepicker_format_datetime = MM/DD/YYYY hh:mm
```

Comment:
I was sure this would have concerns, i've tried several options and none seems to work with Apache Wicket.

Finally i've implemented the same solution as GradebookNG:
https://github.com/sakaiproject/sakai/blob/master/gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/AddOrEditGradeItemPanelContent.java#L114